examples/cfgSync/rollup.config.js

```diff
@@ -0,0 +1,100 @@
+import nodeResolve from "@rollup/plugin-node-resolve";
```

Comment:
This will need to be updated once Siyu's change is submitted, I'm thinking we wait until that one is added before pushing this one in.